airbase/airtable.py

```diff
@@ -257,8 +260,13 @@ async def get_tables(self) -> Optional[List]:  # noqa: F821
 
     async def get_table(self, value: str, key: Optional[str] = None):
         assert key in (None, "id", "name")
+
+        if key == "name":
```

Comment:
`get_table` should first check for attr `tables` in self to allow search by `name` in `self._tables_by_name` which will return a `Table` class with more metadata as attributes